src/Compilers/Core/MSBuildTaskTests/CopyRefAssemblyTests.cs

```diff
@@ -12,6 +12,7 @@
 using Roslyn.Test.Utilities;
 using Microsoft.CodeAnalysis.BuildTasks.UnitTests.TestUtilities;
 using Microsoft.CodeAnalysis.Test.Utilities;
+using ICSharpCode.Decompiler;
```

Comment:
Is this used?

Reply:
Thanks. Not sure how that using snuck in